src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -338,7 +393,30 @@ function MoneyRequestParticipantsSelector({
 
     const shouldShowReferralBanner = !isDismissed && iouType !== CONST.IOU.TYPE.INVOICE && !shouldShowListEmptyContent;
 
+    const goToSettings = useCallback(() => {
```

Comment:
Unfortunately, the number of times this prompt can be shown is unlimited. If the user rejects it, we can show it one more time on iOS and two more times on Android. After that, it won't be shown again.
That's why I'm navigating to settings here, as the user has most likely denied it.
Ah, yeah, in some cases, it can still be triggered. I'll add this so that, if possible, it will show the permission prompt.

Reply:
@perunt If possible, we can display this modal. But if the user has already denied, let's go to setting pages